pkg/virtctl/virtctl.go

```diff
@@ -54,3 +61,13 @@ func (o *Options) Usage() string {
 	usage += o.FlagSet().FlagUsages()
 	return usage
 }
+
+func GetKubevirtClient(flags *flag.FlagSet) (kubecli.KubevirtClient, error) {
```

Comment:
The function is not used right now ...

Reply:
Good catch. Since we've committed to an entirely different direction, it will never be used.